packages/usa-date-picker/src/index.js

```diff
@@ -2230,12 +2248,6 @@ if (!isIosDevice()) {
     [CALENDAR_DATE_CURRENT_MONTH]() {
       handleMouseoverFromDate(this);
     },
-    [CALENDAR_MONTH]() {
-      handleMouseoverFromMonth(this);
-    },
-    [CALENDAR_YEAR]() {
-      handleMouseoverFromYear(this);
-    },
```

Comment:
Removed mouseover from month and year functions because we do not want to re-render and focus these buttons on hover.